app-development-wg/charter/README.md

```diff
@@ -0,0 +1,120 @@
+
+
+## Mission
+
+The following sections describe the focus of the working group, its core responsabilities and the areas that are out of scope. 
+
+## Responsibilities & Deliverables
+
+Cloud-native developers and architects face a double threat: fragmented tooling landscapes and limited awareness of cross-language solutions. This combination creates a time-consuming roadblock – wasting valuable effort to find the right tools for common programming challenges.
+
+This working group aims to empower cloud-native developers and architects by identifying and promoting common practices and tooling for enhancing the inner development loop. One of the objectives will be to provide an overview of categorized tools. Each tool, best practice, and standard the working group covers will highlight real-world case studies, suggestions, and content to help teams move faster. The key categories are below:
+
+- Development Environment:
+  - Code Editors & IDEs (including Cloud-based options)
+  - Plugins & Extensions
+- Build & Automation (Local):
+  - Build Tools
+  - Containerization Tools
+- Testing & Debugging:
+  - Local & Remote Testing Tools
+  - Debugging Tools (including Profiling)
+- Code Quality & Standards:
+  - Linters & Formatters
+  - Static Code Analysis Tools
+  - API Standards & Compliance Tools
+- Cloud Native Development:
+  - APIs & SDKs/CLIs for Cloud Services
+  - Standards that developers can adopt
+  - Polyglot Development Support Tools
+
+
+The following diagram illustrates common tools, tasks, and artifacts in developer workflows.
+
+![](imgs/developer-workflows.png)
+
+In addition to uncovering tools, this group should champion best practices for building developer experiences (DevEx) that enhance productivity and minimize cognitive load. This includes promoting tools that integrate seamlessly within existing workflows and empowering teams to design custom interfaces tailored to domain-specific needs.
+
+### Areas In-Scope
+
+A core objective of this group is to identify and promote existing tools, standards, and practices within the CNCF ecosystem. By showcasing how these elements work together, we aim to map how existing tools and standards can enhance and complement practices and tools already familiar to developers and architects.
```

Comment:
```suggestion
A core objective of this group is to identify and promote existing tools, specifications, and practices within the CNCF ecosystem. By showcasing how these elements work together, we aim to map how existing tools and standards can enhance and complement practices and tools already familiar to developers and architects.
```
CNCF is not a standards body, specifications are how we gain alignment, consistency, and exchange between tools and projects.

Reply:
Just to be clear, CNCF doesn't suggest or represent any (industrial/official) standards, right? If so, any WGs can't propose or suggest standards (from the terminology standpoint), can they?

Reply:
To my knowledge, @caniszczyk please correct me, CNCF does not represent in standards bodies. Regarding proposing or suggesting, some standards are for fees, some are industry specific (payment transactions for instance) so while we can enable adopters and projects to meet them by our understanding of those standards, proposing something like "cloud native projects should meet PCI/DSS" would be out of scope, but adopting a best practice like "secure software development" would be. 

Now, all this being said, if a project wants to provide a mechanism to support adopters in meeting a standard they can, and our groups are welcome to identify requirements of certain industry to help project reach adoption, referencing a standard to do so - but we don't create them.

If Chris has other opinions or perspective I'm keen to update my understanding.